worker-markup/src/main/java/com/hpe/caf/worker/markup/MarkupHeadersAndBody.java

```diff
@@ -209,11 +275,12 @@ private static void addCondensedHeader(Parser nattyParser, Element headersElemen
      * @param headersElement the header element to add headers to.
      * @param lines the lines remaining of the email.
      * @param line the current line containing the header.
+     * @param valueToSplitOn the value that the line parameter should be split on.
      */
-    private void addStandardisedHeader(Parser nattyParser, Element headersElement, String[] lines, String line)
+    private void addStandardisedHeader(Parser nattyParser, Element headersElement, String[] lines, String line, String valueToSplitOn)
```

Comment:
I'm concerned that this functionality doesn't work with this new functionality as it stands.